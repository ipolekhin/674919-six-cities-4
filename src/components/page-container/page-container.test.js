import React from "react";
import renderer from "react-test-renderer";
import PageContainer from "./page-container.jsx";

const offersOfTown = [];

describe(`PageContainer Test`, () => {
  it(`Render PageContainer`, () => {
    const tree = renderer
      .create(
          <PageContainer
            offersOfTown = {offersOfTown}
            renderContainer={() => (
              <div className="page page--gray page--main"></div>
            )}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
