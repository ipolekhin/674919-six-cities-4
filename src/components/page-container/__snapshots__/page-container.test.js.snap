// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PageContainer 1`] = `
<div
  className="page page--gray page--main"
/>
`;
